spring-multirabbit/src/main/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitProperties.java

```diff
@@ -17,6 +17,11 @@
 @ConfigurationProperties(prefix = "spring.multirabbitmq")
 public class MultiRabbitProperties {
 
+    /**
+     * Enables Multiple Rabbit.
+     */
+    private boolean enabled = false;
```

Comment:
This flag was only introduced at v2.4.
For previous versions, it was always enabled.

Reply:
Hi @rwanderc, yes that is correct. I've removed it. 
Thanks 👍🏻